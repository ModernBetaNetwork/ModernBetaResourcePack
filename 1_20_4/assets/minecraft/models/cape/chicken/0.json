{
	"parent": "minecraft:cape/cape",
	"textures": {
		"0": "item/capes/chicken",
		"particle": "item/capes/chicken"
	}
}