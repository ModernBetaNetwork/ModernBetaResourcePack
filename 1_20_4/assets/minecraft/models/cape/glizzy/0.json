{
	"parent": "minecraft:cape/cape",
	"textures": {
		"0": "item/capes/glizzy",
		"particle": "item/capes/glizzy"
	}
}