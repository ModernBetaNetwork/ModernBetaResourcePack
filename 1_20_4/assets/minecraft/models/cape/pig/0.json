{
	"parent": "minecraft:cape/cape",
	"textures": {
		"0": "item/capes/pig",
		"particle": "item/capes/pig"
	}
}