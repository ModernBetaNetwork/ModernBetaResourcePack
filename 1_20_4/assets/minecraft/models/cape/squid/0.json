{
	"parent": "minecraft:cape/cape",
	"textures": {
		"0": "item/capes/squid",
		"particle": "item/capes/squid"
	}
}