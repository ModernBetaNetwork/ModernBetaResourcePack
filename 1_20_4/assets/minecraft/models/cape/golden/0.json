{
	"parent": "minecraft:cape/cape",
	"textures": {
		"0": "item/capes/golden",
		"particle": "item/capes/golden"
	}
}